deeprank/features/FullPSSM.py

```diff
@@ -180,7 +180,7 @@ def get_feature_value(self, cutoff=5.5):
         # get interface contact residues
         # ctc_res = {"A":[chain 1 residues], "B": [chain2 residues]}
         ctc_res = sql.get_contact_residues(cutoff=cutoff)
-        sql.close()
```

Comment:
The `pssm_res_set` contains pssm for all residues, and `ctc_res_set` consider only interface residues, so this condition should always be True.
Also `pssm_res_wo_ctc = pssm_res_set.difference(ctc_res_set)` misstreats the non-interface residues as residues with missing atoms.